indico/util/string.py

```diff
@@ -190,18 +190,23 @@ def remove_tags(text):
     return remove_extra_spaces(pattern.sub(' ', text))
 
 
-def render_markdown(text, **kwargs):
+def render_markdown(text, escape_latex_math=True, **kwargs):
     """ Mako markdown to html filter """
-    math_segments = []
-    placeholder = u"\uE000"
+    if escape_latex_math:
+        math_segments = []
+        placeholder = u"\uE000"
 
-    def math_replace(m):
-        math_segments.append(m.group(0))
-        return placeholder
+        def _math_replace(m):
+            math_segments.append(m.group(0))
+            return placeholder
+
+        text = re.sub(r'\$[^\$]+\$|\$\$(^\$)\$\$', _math_replace, to_unicode(text))
```

Comment:
What do you think of this approach?
https://gist.github.com/nop33/0201a6b952a1f8a6dd4a